e2e/ansible/playbooks/utils/copy_task.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Copy the chaoskube specs to kubemaster
+  copy:
+    src: "{{ item }}"
+    dest: "{{ result_kube_home.stdout }}"
+  delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
```

Comment:
Would it be better to parameterize the delegation as well ? The copy task may not be delegated to master always?

Reply:
Sure, even i gave a taught on this.